Felienne/hedy
# Add different error message for server errors on all auth functions

**Description**

- Adds a specific error message for all auth operations in the client (signup, login, change password, edit profile, recover/reset password.

**Fix for**

- Closes #1164 

**How to test**

- Add a generic error at the top of (say) the signup method (https://github.com/Felienne/hedy/blob/main/website/auth.py#L196). Something like `raise Exception('foo')` would work. This will trigger a 500 error when attempting to sign up.
- Attempt to sign up and notice that the error will contain a text referring to a *server* error.

Note: this PR adds both a code change and changes to the YAML texts because one part needs the other in order to be useful.
